src/AutoRest.CSharp/LowLevel/Output/Samples/DpgOperationSample.cs

```diff
@@ -138,7 +153,28 @@ private Dictionary<string, InputExampleParameterValue> EnsureParameterValueMappi
                     // if this is a required parameter and we did not find the corresponding parameter in the examples, we put the null
                     if (!parameter.IsOptionalInSignature)
                     {
-                        result.Add(parameter.Name, new InputExampleParameterValue(parameter, Null.CastTo(parameter.Type)));
+                        /*
+                         * TODO: remove this when example can contain client parameter value. Following workaround will not needed.
+                         */
+                        ValueExpression parameterExpression = parameter.Type.IsValueType && !parameter.Type.IsNullable ? Default.CastTo(parameter.Type) : Null.CastTo(parameter.Type);
```

Comment:
how about we simplify this just write the `default`?
`default` also works for reference types.